_ml-commons-plugin/api.md

```diff
@@ -0,0 +1,712 @@
+---
+layout: default
+title: API
+has_children: false
+nav_order: 99
+---
+
+# ML Commons API 
+
+---
+
+#### Table of contents
+- TOC
+{:toc}
+
+
+---
+
+The Machine Learning (ML) commons API lets you train ML algorithms synchronously and asynchronously, and then store that model in an ML model index.
```

Comment:
As this is the overall description, should we also add more sentences to describe other main 2 functions
1) predict with the trained model
2) or train and predict with the same data set